### ISSUE TITLE
Autoplot on join

### ISSUE THREAD

**Issue by N0tMyFaultOG** [IntellectualSites/PlotSquaredSuggestions#81]
**PlotSquared Feature Suggestion:**
We should add a config option to auto-claim a plot on the first join. I've seen a few people asking for it. It might be useful for some servers.

**How should it be implemented?**
Having a boolean config option whether to allow or deny the named behavior.

**Additional context**
/

**Checklist**:

- [X] I made sure there are no duplicates of my suggestion [(Use search)](https://github.com/IntellectualSites/PlotSquaredSuggestions/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+)
- [X] I made sure my suggestion does not exist yet [(Check the wiki)](https://github.com/IntellectualSites/FastAsyncWorldEdit-1.13/wiki)
- [X] I made sure my suggestion is useful for the majority of PlotSquared users

**Comment by Sauilitired** [comment]
Yeah, I thought about this today lol. It'd be especially useful for servers with no dedicated spawn plot

**Comment by AeSix** [comment]
It's easy enough to write a commandhelper script for this...
BUT it would definitely be easier for people who can't use CH... or don't want to.  Can't think of why it'd be difficult to impliment either?

**Comment by RedstoneFuture** [comment]
In the other hand with a first-join plugin (e.g. FirstJoinPlus) you can define commands for the first server-spawn too.

**Comment by RedstoneFuture** [comment]
Moreover, you need a plugin like FirstJoinPlus too, because you must give a new player a instruction message, starter kit or other kind of setup. Without any start help you can't teleport the player to her plot, I think.
